.github/fly-wsproxies/paris-coder.toml

```diff
@@ -21,6 +22,6 @@ primary_region = "cdg"
   min_machines_running = 0
```

Comment:
2 servers in paris

Reply:
I have set 2 in each using `fly scale count 2`
You can verify with `fly scale show`
This is for auto stopping when no one is using them